pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: typed_data
-version: 1.1.5
+version: 1.1.6
```

Comment:
Since this doesn't change any user-consumable code, can we just make it 1.1.6-dev?

Reply:
done.

Reply:
Done?

Reply:
bumped version and updated sdk constraints. does this look ok?